source/slang/slang-parameter-binding.cpp

```diff
@@ -329,11 +329,18 @@ struct UsedRanges
     }
 };
 
+struct ParameterAndLayoutConstants
+{
+    static const uint32_t kInputAttachmentIndexUnused = 0xFFFFFFFF;
```

Comment:
probably you should make this
```
static const constexpr
```
Don't instantiate a static member variable inside of struct declaration.
You need to either make it constexpr or instantiate it outside of struct declaration.